sematic/examples/bert_fine_tune_scam/BUILD

```diff
@@ -0,0 +1,13 @@
+sematic_example(
```

Comment:
Let's call this example `bert_fine_tuning`. Can you rename the directory and this Bazel target?